orion/controllers/achieve/achieveController.go

```diff
@@ -0,0 +1,108 @@
+package achieve
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetAchievements(c *gin.Context) {
+	// Query Repo
+	achieveList := GetAllAchievements()
+
+	// JSON Response
+	c.JSON(http.StatusOK, achieveList)
+	return
+}
+
+func GetAchievement(c *gin.Context) {
+	// Incoming parameters
+	id64, _ := strconv.ParseUint(c.Param("id"), 10, 32)
```

Comment:
nice! Though I think if we get an error, we should probably do something about it instead of continuing.
For now, you can use `panic(err)`.

We will probably do something better to handle errors in the near future.